backend-dotnet/Meowmentum.Server.Dotnet.Api/Meowmentum.Server.Dotnet.Api/Controllers/AuthController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Meowmentum.Server.Dotnet.Business.Abstractions;
+using Meowmentum.Server.Dotnet.Shared.Requests.Registration;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Meowmentum.Server.Dotnet.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class AuthController : ControllerBase
```

Comment:
You can simply pass AuthController(IAuthService authService) without initializing with constructor.